spring-kafka/src/main/java/org/springframework/kafka/listener/adapter/AbstractDelegatingMessageListenerAdapter.java

```diff
@@ -35,16 +38,20 @@
  * @since 1.1
  *
  */
-public abstract class AbstractMessageListenerAdapter<K, V, T> implements ConsumerSeekAware {
+public abstract class AbstractDelegatingMessageListenerAdapter<K, V, T>
```

Comment:
IDEA says me that we don't need `K, V`:
![no_need_kv](https://cloud.githubusercontent.com/assets/1071590/24884908/bd590a00-1e19-11e7-850d-8329af2bd0b2.png)